### DISCUSSION TITLE
Could this setup be adapted for InDesign?

### DISCUSSION THREAD

**Discussion by arcsaber** [YoungFellow-le/photoshop-22-linux#9] [Ideas]
Hello,

Do you think that your setup could be adapted for InDesign? Would be amazing.

Cheers

**Comment by albakhtari** [comment]
Hey there,

The quick answer to your question: _I have no idea :man_shrugging:_

This is because I'm not the one who found the dependencies for Photoshop, it was @MiMillieuh who did it, and I don't really have time to do so.
You can ask them, and if they get it to work, then I might create an installer for it (if I have time).
